packages/core/src/di/injector.ts

```diff
@@ -113,7 +114,7 @@ export abstract class Injector {
    * @internal
    * @nocollapse
    */
-  static __NG_ELEMENT_ID__ = -1;
+  static __NG_ELEMENT_ID__ = DIBloomFilterIds.InjectorElementId;
```

Comment:
I think we can omit the `ElementId` part:
```suggestion
  static __NG_ELEMENT_ID__ = DIBloomFilterIds.Injector;
```

Reply:
Updated